pymc3/distributions/continuous.py

```diff
@@ -991,6 +990,25 @@ def __init__(self, distribution, lower, upper, *args, **kwargs):
         if hasattr(self.dist, 'mode'):
             self.mode = self.dist.mode
 
+        if transform == 'infer':
+
+            default = self.dist.default()
```

Comment:
Added a very basic strategy to avoid testvals outside the definition interval of bounded distributions. I think this is a good place to check for the consistency but there might be a better place.